### PR TITLE
PROD-776: Update authentication_classes to use JWTAuthentication

### PR DESCRIPTION
**Pull request by azarembok**
This is needed so that veda can authenticate with DOT auth.

### CODE REVIEW

File: edxval/views.py [hunk 1]
```diff
@@ -75,7 +77,7 @@ class VideoList(generics.ListCreateAPIView):
     """
     GETs or POST video objects
     """
-    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    authentication_classes = (JwtAuthentication, OAuth2Authentication, SessionAuthentication)
```
Comment (doctoryes) [inline edxval/views.py]: Why is the `OAuth2Authentication` being left in this list? Isn't `JwtAuthentication` sufficient?

Comment (azarembok) [reply edxval/views.py]: @doctoryes I mistakenly thought it was still required for the tests to pass, but it turns out that they only need SessionAuthentication to be there.  So I have now removed OAuth2Authentication.

**Comment by timmc-edx** [comment]
This appears to have been the last reference to `rest_framework_oauth` in this repo—does that mean that the `djangorestframework-oauth` dependency can be removed now?

**Comment by azarembok** [comment]
> This appears to have been the last reference to `rest_framework_oauth` in this repo—does that mean that the `djangorestframework-oauth` dependency can be removed now?

Yes, it looks like that can be removed.

**Pull request merged**